Typo "Pytorch"→"PyTorch"

hyoshioka0128 opened the pull request:
https://learn.microsoft.com/en-us/windows/ai/windows-ml/tutorials/pytorch-analysis-deploy-model 
https://github.com/MicrosoftDocs/windows-ai-docs/blob/docs/docs/windows-ml/tutorials/pytorch-analysis-deploy-model.md 
#PingMSFTDocs

opbld34 commented:
Docs Build status updates of commit _[2e218ed](https://github.com/MicrosoftDocs/windows-ai-docs/pull/125/commits/2e218eded068be82956121a9b69c3bb17920a056)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/vision-skills-diagram-wide.png](https://github.com/MicrosoftDocs/windows-ai-docs/blob/elcowle-master-links/docs/images/vision-skills-diagram-wide.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/windows-vision-skills/index?branch=pr-en-us-125) |
[docs/images/vision-skills-diagram2-wide.png](https://github.com/MicrosoftDocs/windows-ai-docs/blob/elcowle-master-links/docs/images/vision-skills-diagram2-wide.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/images/vision-skills-diagram2-wide.png?branch=pr-en-us-125) |
[docs/windows-vision-skills/index.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/elcowle-master-links/docs/windows-vision-skills/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/windows-vision-skills/index?branch=pr-en-us-125) |
[docs/windows-vision-skills/tutorial2.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/elcowle-master-links/docs/windows-vision-skills/tutorial2.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/windows-vision-skills/tutorial2?branch=pr-en-us-125) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C18%5C28ddff79-5366-c51c-8202-1b9e8beacb23%5CPullRequest%5C201904182242265861-125%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=euBH2JJDkFyyFgKUOHB6gqPcB%2BZgyAakiyZJXymqXEo%3D&st=2019-04-18T22%3A39%3A00Z&se=2019-05-19T22%3A44%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

hyoshioka0128 committed:
Typo "Pytorch"→"PyTorch"

https://learn.microsoft.com/en-us/windows/ai/windows-ml/tutorials/pytorch-analysis-deploy-model
https://github.com/MicrosoftDocs/windows-ai-docs/blob/docs/docs/windows-ml/tutorials/pytorch-analysis-deploy-model.md
#PingMSFTDocs
